goinstant/features/service_commands.feature

```diff
@@ -0,0 +1,16 @@
+Feature: Test Deploy Commands
+  Scenario: Initialise Core Service
+    When the command "init core -t=docker" is run
```

Comment:
Specifying the target as docker should no longer be required here ([bugfix](https://github.com/openhie/instant/pull/180/files) has been merged), so I think we should have some tests using it and some not, to ensure we cover that code as well?

Reply:
This would only work if we were calling the commands through the go functions. But, what actually happens is that we take that command and pass it through to a cloned version of the function runCommand (named runTestCommand in godogs_test.go), thereby bypassing extractCommands (where the default target launcher is checked)